packages/components/src/list/collapsible-list/index.tsx

```diff
@@ -0,0 +1,109 @@
+/**
+ * External dependencies
+ */
+import { Dashicon } from '@wordpress/components';
+import { useState, useCallback, Children } from '@wordpress/element';
+import { Transition } from 'react-transition-group';
+
+/**
+ * Internal dependencies
+ */
+import { ExperimentalListItem } from '../experimental-list-item';
+import { ListProps, ExperimentalList } from '../experimental-list';
+
+type CollapsibleListProps = {
+	hideText: string;
+	showText: string;
+	collapsed?: boolean;
+	minChildrenToShow?: number;
```

Comment:
How about `itemsToShow` or `hideItemsAfter`.. or even just `show`?

Reply:
Yeah it's a bit much, I end up going with `show`.